extensions/ui/org.eclipse.smarthome.ui.paper/pom.xml

```diff
@@ -59,10 +59,10 @@
           <execution>
             <id>gulp build</id>
```

Comment:
I couldn't decide to rename it as actually it's still the "gulp build" that is run and I felt uncertain if that follows any convention. That said, I'd be happy to rename that to `frontend build` or whatever you prefer.